srcpkgs/azote/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'azote'
+pkgname=azote
+version=1.1.1
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="azote"
+hostmakedepends="python3-setuptools"
+depends="python3>=3.5 python3-setuptools python3-gobject python3-Pillow gtk+3 feh xrandr wmctrl"
+short_desc="Wallpaper manager for Sway, i3 and some other WMs"
+maintainer="Piotr Miller <nwg.piotr@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/nwg-piotr/azote"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=f7c818b11d3add7af9388589fc6658b0121772acdf899a5ba4a275d082621a63
+
+do_install() {
+	python3 setup.py install --prefix=/usr --root=${DESTDIR} ${make_install_args}
```

Comment:
This line is exactly what `do_install` does in the build_style anyways. It would be better to use the build_style function and put everything that's not covered by the default function in a `post_install` function.